doc/data/messages/n/nan-comparison/bad.py

```diff
@@ -0,0 +1,5 @@
+import numpy
```

Comment:
```suggestion
import numpy as np
```

Reply:
I know `as np` is customary but it's not necessary to demonstrate nan-comparison...

Reply:
Sure, but if we have numpy example we might as well have the same style than the numpy doc.